.travis.yml

```
@@ -23,4 +23,4 @@ script:
 - if [ "$TYPE" = "build" ]; then python -m compileall ./musicbot; fi;
 
 after_success:
-- if [ "$TRAVIS_PULL_REQUEST" == "false" ] && [ "$TYPE" == "docker" ]; then docker login -u "$DOCKER_USERNAME" -p "$DOCKER_PASS"; docker tag musicbot justsomebots/musicbot:$TRAVIS_BRANCH; docker push justsomebots/musicbot; fi;
+- if [ "$TRAVIS_PULL_REQUEST" == "false" ] && [ "$TYPE" == "docker" ]; then docker login -u "$DOCKER_USERNAME" -p "$DOCKER_PASS"; docker tag musicbot $DOCKER_USERNAME/musicbot:$TRAVIS_BRANCH; docker push $DOCKER_USERNAME/musicbot; fi;
```

Please change this back:
```suggestion
- if [ "$TRAVIS_PULL_REQUEST" == "false" ] && [ "$TYPE" == "docker" ]; then docker login -u "$DOCKER_USERNAME" -p "$DOCKER_PASS"; docker tag musicbot justsomebots/musicbot:$TRAVIS_BRANCH; docker push justsomebots/musicbot; fi;
```
`$DOCKER_USERNAME` would not work for the actual repo URL.

---

I changed it for testing and forget to change it back. apologies.